application/models/policy.go

```diff
@@ -388,13 +388,13 @@ func scopeSearchPolicies(searchText string) pop.ScopeFunc {
 	return func(q *pop.Query) *pop.Query {
 		return q.Where("policies.id IN ("+
```

Comment:
Would this be better? (And does it still work?)
```suggestion
		return q.RawQuery("policies.id IN ("+
```

Reply:
No, it doesn't work. It appears to replace the entire query with only the text passed to RawQuery.

Reply:
The function doc for RawQuery:
```
// RawQuery will override the query building feature of Pop and will use
// whatever query you want to execute against the `Connection`. You can continue
// to use the `?` argument syntax.
```